scan/lib/scan/options.rb

```diff
@@ -146,6 +146,11 @@ def self.available_options
                                      type: Boolean,
                                      default_value: false,
                                      optional: true),
+        FastlaneCore::ConfigItem.new(key: :suppress_xcode_output,
+                                     env_name: "SUPPRESS_OUTPUT",
```

Comment:
Can we change this to be `SUPPRESS_XCODE_OUTPUT` to make it match the key? 😊

Reply:
Most probably I copied it from another place and forgot to update it 😃